data/blog/electron-8-0.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Electron 8.0.0
+author:
+- jkleinsc
+- sofianguy
+date: '2020-02-04'
+---
+
+Electron 8.0.0 has been released! It includes upgrades to Chromium `80`, V8 `8.0`, and Node.js `12.13.0`. We've added Chrome's built-in spellchecker, and much more!
+
+---
+
+The Electron team is excited to announce the release of Electron 8.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). The release is packed with upgrades, fixes, and new features. We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+* Chromium `80.0.3987.86`
+    * [New in Chrome 79](https://developers.google.com/web/updates/2019/12/nic79)
+    * [New in Chrome 80](TODO)
+* Node.js `12.13.0`
+    * [Node 12.13.0 blog post](https://nodejs.org/en/blog/release/v12.13.0/)
+* V8 `8.0`
+    * [V8 7.9 blog post](https://v8.dev/blog/v8-release-79)
+    * [V8 8.0 blog post](https://v8.dev/blog/v8-release-80)
+
+### Highlight Features
+* Implemented usage of Chrome's built-in spellchecker feature. See more details in [#20692](https://github.com/electron/electron/pull/20692) and [#21266](https://github.com/electron/electron/pull/21266).
+
```

Comment:
I agree with @cruzerld - the third-party cookie issue is likely going to affect several apps, I think it's worth mentioning. 

This is just an opinion, but I also think it's worth adding the new structured clone algorithm to Highlight Features in addition to including it in breaking changes. It's a pretty impressive performance improvement! I think folks will be interested in that 🙂 

```suggestion
* IPC communication now uses v8's Structured Clone Algorithm. This is faster, more featureful, and less surprising than the existing logic, and brings about a 2x performance boost for large buffers and complex objects. Latency for small messages is not significantly affected. See more details in [#20214](https://github.com/electron/electron/pull/20214).
```